### PR TITLE
COMMERCE-11644 Show exact dateTime for date field

### PR DESCRIPTION
**Pull request by brianikim**
@andrea-ale-sbarra ,

https://issues.liferay.com/browse/COMMERCE-11644, tied to LPP

Sincerely,
Brian I. Kim

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
@andrea-ale-sbarra , made new change to handle itemSelector view for CPDefinitionLink

File: modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/definition_item_selector.jsp [hunk 1]
```diff
@@ -92,10 +94,10 @@ String itemSelectedEventName = cpDefinitionItemSelectorViewDisplayContext.getIte
 				value="<%= HtmlUtil.escape(cpDefinitionItemSelectorViewDisplayContext.getSku(cpDefinition, locale)) %>"
 			/>
 
-			<liferay-ui:search-container-column-date
+			<liferay-ui:search-container-column-text
 				cssClass="table-cell-expand"
 				name="modified-date"
-				property="modifiedDate"
+				value="<%= HtmlUtil.escapeJS(dateFormatDateTime.format(cpDefinition.getModifiedDate())) %>"
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/definition_item_selector.jsp:100]: Can you please wrap this into a new method of the dispaly context?

Thank you!

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/definition_item_selector.jsp (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/definition_item_selector.jsp]: It should be good now. See new method @andrea-ale-sbarra

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM, thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:stop

**Comment by andrea-ale-sbarra** [comment]
ci:test:stop

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
New PR: #3860

**Comment by pietrobocale** [comment]
ci:close

**Pull request closed without merging**